pages/blog.js

```diff
@@ -0,0 +1,115 @@
+import {
+  Footer,
+  Header,
+  Entry,
+  Author,
+  Feature,
+  Pagination,
+  Topic,
+} from "@landing";
+
+import posts from "data/blog.json";
+import features from "data/feature.json";
+
+const Blog = () => (
+  <>
+    <Header />
+
+    <div class="relative overflow-x-hidden">
+      <div class="px-6 py-8">
+        <div class="container flex justify-between mx-auto">
+          <div class="w-full lg:w-8/12">
+            <div class="flex items-center justify-between">
+              <h1 class="ml-20 text-center font-bold text-dark text-5xl">
+                <span className="text-purple-500 text-2xl text-5xl mr-1">
+                  |
+                </span>
+                Blog
+              </h1>
+              <div className="mr-16">
+                <select class="w-full border-gray-300 rounded-md shadow-sm focus:border-indigo-300 focus:ring focus:ring-indigo-200 focus:ring-opacity-50">
+                  <option> Latest </option>
+                  <option> Oldest </option>
+                </select>
+              </div>
+            </div>
+
+            <div class="ml-20">
+              {posts.map((entry) => (
+                <Entry {...entry} />
+              ))}
+            </div>
+
+            <Pagination />
+          </div>
+
+          <div class="hidden w-4/12 lg:block">
+            <div class="px-8 mt-10">
+              <h1 class="text-xl font-bold text-dark">
+                ⭐ Artigos em Destaque
+              </h1>
+              {features.map((entry) => (
+                <Feature {...entry} />
+              ))}
+            </div>
+
+            <div class="px-8">
+              <h1 class="mt-8 mb-4 text-xl font-bold text-dark">✍️ Autores</h1>
+              <div class="mt-4 flex flex-col max-w-sm px-6 py-2 bg-white rounded-lg shadow-md">
+                <ul class="-mx-4">
+                  <Author
+                    author="Filipe Felício"
+                    photo="filipe.png"
+                    numberPosts={2}
+                  />
+                  <Author
+                    author="Nelson Estevão"
+                    photo="nelson.png"
+                    numberPosts={2}
+                  />
+                  <Author
+                    author="Marta Silva"
```

Comment:
I think name would be a better prop here. What do you think?

Reply:
Yeah! Why not :yin_yang: